packages/calling-component-bindings/src/utils/participantListSelectorUtils.ts

```diff
@@ -50,3 +52,56 @@ const convertRemoteParticipantToParticipantListParticipant = (
       getIdentifierKind(identifier).kind === 'communicationUser' || getIdentifierKind(identifier).kind === 'phoneNumber'
   };
 };
+
+/* @conditional-compile-remove(rooms) */
+/**
+ * @private
+ */
+export const memoizedConvertAllremoteParticipantsBeta = memoizeFnAll(
+  (
+    userId: string,
+    displayName: string | undefined,
+    state: 'Idle' | 'Connecting' | 'Ringing' | 'Connected' | 'Hold' | 'InLobby' | 'EarlyMedia' | 'Disconnected',
+    isMuted: boolean,
+    isScreenSharing: boolean,
+    isSpeaking: boolean,
+    role: Role
+  ): CallParticipantListParticipant => {
+    return convertRemoteParticipantToParticipantListParticipantBeta(
+      userId,
+      displayName,
+      state,
+      isMuted,
+      isScreenSharing,
+      isSpeaking,
+      role
+    );
+  }
+);
+
+/* @conditional-compile-remove(rooms) */
+const convertRemoteParticipantToParticipantListParticipantBeta = (
+  userId: string,
+  displayName: string | undefined,
+  state: 'Idle' | 'Connecting' | 'Ringing' | 'Connected' | 'Hold' | 'InLobby' | 'EarlyMedia' | 'Disconnected',
+  isMuted: boolean,
+  isScreenSharing: boolean,
+  isSpeaking: boolean,
+  role: Role
+): CallParticipantListParticipant => {
+  const identifier = fromFlatCommunicationIdentifier(userId);
+  return {
+    userId,
+    displayName,
+    state,
+    isMuted,
+    isScreenSharing,
+    isSpeaking,
+    // ACS users can not remove Teams users.
+    // Removing unknown types of users is undefined.
+    isRemovable:
```

Comment:
avoid duplicate logic, if conditional compile isn't working, instead have convertRemoteParticipantToParticipantListParticipantBeta call convertRemoteParticipantToParticipantListParticipant then apply the beta adaptions on top:
```
convertRemoteParticipantToParticipantListParticipantBeta = (args) => {
  return {
    convertRemoteParticipantToParticipantListParticipant(args),
    // beta things here
  }
}
```

Reply:
This will also highlight what differs between these two functions without having to compare every line

Reply:
However, we should be updating our conditional compile to be able to work for this util function

Reply:
Good point. Thanks.